backend/docs/developers/setup-local-env.rst

```diff
@@ -43,19 +44,36 @@ You need the following libraries and/or programs:
        npm run build
 
 #. Run the migrations with ``src/manage.py migrate``
+
+.. note::
+
+       If you get an error about the ``django.contrib.gis`` module,
```

Comment:
Does it necessarily need superuser permissions or just any adequate permissions, please check with @SilviaAmAm

Reply:
Yes you need to be a superuser to install the postgis extension. You can either log into postgres as superuser, connect to the database and run `CREATE EXTENSION POSTGIS` or you can give superuser rights to the openarchiefbeheer user. I generally do the former.